test/fixtures/mountains.yml

```diff
@@ -4,16 +4,16 @@ Fuji:
   terrain_diff: ★★
   physical_diff: ★★
   length: '日帰り'
-  region_id: 1
   lat: 35.360747509167
   lng: 138.72735534793
+  region_id: 1
 
 Fake_Mountain:
   name: 'フェーク'
   altitude: 1111
   terrain_diff: ★★
   physical_diff: ★★
   length: '一泊二日'
-  region_id: 2
   lat: 35.360747509167
-  lng: 138.72735534793
\ No newline at end of file
+  lng: 138.72735534793
+  region_id: 1
```

Comment:
missing newline

Reply:
I don't understand where the newline should be or why.